docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,242 @@ grand_parent: "General"
 ---
 
 # Getting Started With bpmn-js
+
+bpmn-js is a [BPMN 2.0](https://www.omg.org/spec/BPMN/2.0/About-BPMN/) rendering toolkit and web modeler. It allows to view and edit BPMN 2.0 diagrams in the browser without any backend requirement.
+
+Feel free to play with it [here](https://demo.bpmn.io/s/start).
+
+![image](/assets/images/bpmn-start.gif)
+
+There are two ways to include bpmn-js in your application:
+
+* An [all-in-one pre-packaged version of the library](https://unpkg.com/browse/bpmn-js/dist/) allowing you to quickly add BPMN to any web application.
+* The [npm](http://npmjs.com/package/bpmn-js) version, which is more complicated to set up but gives you access to individual library components and allows for easier extensibility.
+
+## Including the Root Element
+
+Both as a viewer or modeler, bpmn-js requires a root DOM element (a simple DIV) on which the rendering is performed. Simply, create a DIV element in the HTML file:
```

Comment:
At this point in time the viewer does not know about viewer or modeler yet. I would therefore just remove these 6 first words.

Reply:
Done 👍